drivers/amazonec2/amazonec2.go

```diff
@@ -1103,14 +1104,15 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) ([]
 	}
 
 	for _, p := range d.OpenPorts {
-		port, protocol, err := splitPortProto(p)
+		port, protocol := driverutil.SplitPortProto(p)
+		portNum, err := strconv.Atoi(port)
```

Comment:
use strconv.ParseInt, it already returns int64 so you dont have to typecast again below.